### DISCUSSION TITLE
PyCafe - A platform to run, edit and share Python apps - thank you to the team

### DISCUSSION THREAD

**Discussion by maartenbreddels** [pyodide/pyodide#4926] [Show and tell]
Dear devs,

A while ago, we launched https://py.cafe - A platform to run, edit and share Python apps on your browser with 1 click.

This was only possible due to pyodide. It's been a pleasure to see this project evolve, and my (little) interaction with you (https://github.com/pyodide/pyodide/issues/4847, https://github.com/pyodide/micropip/issues/103) has been a pleasure.

We'd love your feedback on this project, and hopefully, you will find it useful as well. We've been using it for https://github.com/widgetti/solara to answer questions with live code, and it's being picked up for integration in docs as well: https://github.com/mckinsey/vizro/pull/569
Since the applications and code can be embedded using iframes.

I hope this project is inspirational for you; to see things being built on top of what you build.
Although we do not earn profits yet, as a gesture, we have sent a $750 sponsorship, but we also hope to contribute back in the form of code or opening issues.

Regards,

the PyCafe team (Mario Buikhuizen, Jack Parmer, Iisakki Rotko and Maarten Breddels)

**Comment by ryanking13** [comment]
Cool, thanks for sharing your project! It's always happy to see people making amazing stuff using Pyodide.

**Comment by agriyakhetarpal** [comment]
I had the pleasure of mentioning PyCafe in my undergraduate thesis – in a section mentioning projects to look out for that make use of Pyodide. It seems like a fantastic platform for distributing code snippets for science and games!

**Reply by maartenbreddels** [reply]
That is great to hear, thanks for the kind words. Would you mind sharing your thesis?

**Reply by agriyakhetarpal** [reply]
Sure, @maartenbreddels. I have to note here that I haven't graduated yet, so I should be able to do so once I do. I shall email you a PDF!
